package.json

```diff
@@ -39,65 +39,56 @@
 		"lodash": "4.*",
 		"pathfinding": "^0.4.18",
 		"paths-js": "^0.4.11",
-		"react": "17.*"
+		"react": "16.* || 17.*"
 	},
 	"devDependencies": {
-		"@babel/core": "^7.13.15",
-		"@babel/preset-react": "^7.13.13",
-		"@emotion/react": "^11.1.5",
-		"@emotion/styled": "^11.3.0",
-		"@storybook/addon-actions": "^6.2.8",
-		"@storybook/addon-options": "^5.3.21",
-		"@storybook/addons": "^6.2.8",
-		"@storybook/react": "^6.2.8",
-		"@storybook/storybook-deployer": "^2.8.7",
-    "@storybook/builder-webpack5": "^6.2.8",
-		"@storybook/theming": "^6.2.8",
-		"@types/dagre": "^0.7.44",
-		"@types/jest": "^26.0.22",
-		"@types/jest-environment-puppeteer": "^4.4.1",
-		"@types/lodash": "^4.14.168",
-		"@types/node": "^14.14.37",
-		"@types/promise": "^7.1.30",
-		"@types/puppeteer": "^5.4.3",
-		"@types/react": "^17.0.3",
+    "emotion": "11.*",
```

Comment:
i would much prefer it if u used a tool such as `ncu` to bump to exact exact versions

Reply:
Use a tool, i don't understand?

Reply:
updated to exact versions.